services/device-registry-base/src/main/java/org/eclipse/hono/service/management/device/Device.java

```diff
@@ -54,6 +54,9 @@
     @JsonFormat(with = JsonFormat.Feature.ACCEPT_SINGLE_VALUE_AS_ARRAY)
     private List<String> memberOf = new LinkedList<>();
 
+    @JsonInclude(value = Include.NON_NULL)
+    private String mapper = null;
```

Comment:
just for reference, in Java you do not need to explicitly initialize with `null` as it is the default. It doesn't harm to make it explicit, though, of course ...